frontend/src/libraries/formatting.ts

```diff
@@ -6,3 +6,10 @@ export function truncateAccountId(
     ? accountId.slice(0, 5) + "…" + accountId.slice(accountId.length - 10)
     : accountId;
 }
+
+export const cumulativeSumArray = (array: Array<number>) =>
```

Comment:
This method has nothing to do with `formatting`, so it should be in a different file. I suggest `stats.ts` (once we will have more functions to add, we may consider turning it into a folder).